packages/expo-haptics/README.md

```diff
@@ -36,6 +36,8 @@ No additional set up necessary.
 
 Run `npx pod-install` after installing the npm package.
 
+### Permission
+
 This module requires permission to control vibration on the device, it's added automatically.
 
 ```xml
```

Comment:
Hey @SnowingFox, thanks for the contribution. 👍 

I think that we should move this fragment under "Android" header instead, like we do it on i.e. `expo-av`:
* https://github.com/expo/expo/blob/main/packages/expo-av/README.md#configure-for-android